Metro-Records/la-metro-dashboard
# Upgrade Airflow to 2.6

## Overview

This PR upgrades Airflow to the latest version (2.6.0).

### Checklist


### Notes

A couple of things not reflected in the title and/or code:

- I changed the way our deployment configs are organized so that they're less annoying to deal with. I also moved deployment secrets into an encrypted .env file and decrypted the Supervisor settings.
- I upgraded Python on the staging server following these instructions: https://cloudbytes.dev/snippets/upgrade-python-to-latest-version-on-ubuntu-linux
- Contrary to @antidipyramid's surgical approach, I dropped the Airflow database before deploying to sidestep those upgrade issues.

## Testing Instructions

These changes have been deployed to the staging site at https://la-metro-dashboard-staging.datamade.us

 * Run `docker-compose build`
 * Run `docker-compose up` and verify the Airflow version at `localhost:8080`

Handles #95
